e2e/layer-fs-smoke-test.sh

```
@@ -396,6 +396,64 @@ start_layer_mount() {
+  fi
+  if is_mounted_path "$mount_point"; then
+    drive9 umount --timeout "$FUSE_UMOUNT_TIMEOUT" "$mount_point" >/dev/null 2>&1
+    echo "layer mount left a mounted path after checkpoint mismatch" >&2
+    return 1
+  fi
+  if ! grep -F "checkpoint $checkpoint_ref belongs to layer" "$mount_log" >/dev/null; then
+    echo "layer mount mismatch log did not mention checkpoint ownership" >&2
+    return 1
+  fi
+  return 0
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)